general-guidelines/validating-api-descriptions.md

```diff
@@ -0,0 +1,74 @@
+# Validating API Description
+
+API description files (e.g. OpenAPI specification for REST APIs) SHOULD be validated to verify whether they comply with this guidelines. In order to do that you can utilize [Spectral](https://github.com/stoplightio/spectral) ruleset implementing them.
+
+## Installing Spectral
+
+To install Spectral you will need Node.js and a package manager (npm or yarn).
+
+```
+npm install -g @stoplight/spectral
+
+# OR
+
+yarn global add @stoplight/spectral
+```
+
+### Docker
+
+Spectral is also available as a Docker image, which can be handy for all sorts of things, like if you're contributing code to Spectral, want to integrate it into your CI build.
+
+```
+docker run --rm -it stoplight/spectral lint "${url}"
+```
+
+If the file you want to lint is on your computer, you'll need to mount the directory where the file resides as a volume
+
+```
+docker run --rm -it -v $(pwd):/tmp stoplight/spectral lint "/tmp/file.yaml"
+```
+
+## Using Spectral
+
+Once installed Spectral you can validate any `YAML` or `JSON` file according to a given set of rules. Spectral has a predefined set of rules validating OpenAPI 2.x (Swagger) and OpenAPI 3.x files.
+
+Spectral comes with a CLI and can be run from your command line:
+
+```
+spectral lint MY_SPEC_FILE.yaml
+```
+
+For more details about how to utilize Spectral CLI you can check the CLI built-in help:
+
+```
+spectral lint -h
+```
+
+or go to the official [Spectral documentation](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/cli.md).
+
+Spectral can also be used from within Javascript. For details on how to accomplish this, please refer to the [documentation](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/javascript.md).
+
+## Validating with Adidas API Guidelines
+
+To check whether your API Specification complies with Adidas API Guidelines copy the `.spectral` file from this repository ([here](https://github.com/adidas/api-guidelines/blob/master/.spectral.yml)).

```

Comment:
I'd suggest against copying the file around. People should always use the latest from the repository. 

Eventually we could create a npm package from this repository so people would do something like `$ npm install -g adidas-api-guidelines@2.3.0` and then `$ api-guidelines  check <oas-file>` ...

Reply:
you want the npm thing now or can we leave the first release as it is and the second iteration will already package it?

Reply:
changed. ill package the guidelines in another PR